app/products/calls/store/actions/calls.ts

```diff
@@ -102,6 +104,7 @@ export function joinCall(channelId: string): ActionFunc {
             ws.disconnect();
             ws = null;
         }
+        dispatch(setSpeakerphoneOn(false));
```

Comment:
This is for the redux side effect, not the actual speakerphone (which starts off in off mode by default, it looks like).